File: albedo-boot-modules/albedo-boot-web-starter/src/main/java/com/albedo/java/modules/test/service/TestTreeService.java
/**
 * Copyright &copy; 2018 <a href="https://github.com/somewhereMrli/albedo-boot">albedo-boot</a> All rights reserved.
 */
package com.albedo.java.modules.test.service;

import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

import com.albedo.java.common.persistence.service.TreeVoService;
import com.albedo.java.modules.test.domain.vo.TestTreeVo;
import com.albedo.java.modules.test.domain.TestTree;
import com.albedo.java.modules.test.repository.TestTreeRepository;
/**
 * 测试树管理Service 测试树管理
 * @author admin
 * @version 2018-03-23
 */
@Service
@Transactional
public class TestTreeService extends TreeVoService<TestTreeRepository, TestTree, String, TestTreeVo>{


}

File: albedo-boot-ui-angular/src/main/webapp/app/theme/pages/modules/test/testTree/testTree.service.ts
/**
 * Copyright &copy; 2018 <a href="https://github.com/somewhereMrli/albedo-boot">albedo-boot</a> All rights reserved.
 */
import { Injectable } from '@angular/core'
import { HttpClient } from "@angular/common/http"
import { TreeService } from "../../../../../shared/base/service/tree.service"
import { PublicService } from "../../../../../shared/base/service/public.service"
import { TestTree } from "./testTree.model"


@Injectable()
export class TestTreeService extends TreeService<TestTree> {

    constructor(
        protected http: HttpClient,
        protected publicService: PublicService) {
        super(http, publicService.getServiceCtx('test_testTree') + '/test/testTree')
    }

}
